src/libaktualizr/utilities/events.h

```diff
@@ -116,4 +115,6 @@ class CampaignCheckComplete : public BaseEvent {
 
 }  // namespace event
 
+using EventChannel = boost::signals2::signal<void(std::shared_ptr<event::BaseEvent>)>;
```

Comment:
A bit weird to define it outside the namespace which contains all other declarations in this file. Should we rename it back to the old `events::Channel`, even if it's not the same type at all?

Reply:
> A bit weird to define it outside the namespace which contains all other declarations in this file.

Agreed. I didn't want to define it here, since nothing else in `events.h` since nothing else in that file has anything to do with signals and such, but I couldn't find a better place to do it.

> Should we rename it back to the old events::Channel, even if it's not the same type at all?

Not a bad idea. Doesn't matter if it isn't the same type; I got rid of the old `Channel` class, so it should be unambiguous after this point.